www/experiments.php

```diff
@@ -554,99 +554,170 @@ function refreshExperimentFormState(){
             echo "$id";
                              } ?>';
         if(priorState && currentTestID && priorState.indexOf("resubmit="+ currentTestID) > -1 ){
-            var form = $("form.experiments_grades");
-            form[0].reset();
+            var form = document.querySelector("form.experiments_grades");
+            form.reset();
             var pairs = priorState.split("&");
 
                 pairs.forEach(pair => {
                     var keyval = pair.split("=");
+                    if( keyval[0]){
+                        keyval[0] = decodeURIComponent(keyval[0]);
+                    }
+                    if( keyval[1]){
+                        keyval[1] = decodeURIComponent(keyval[1]);
+                    }
                     if( keyval[0] !== 'runs' ){
-                        var input = form.find("[name='" + keyval[0] + "']");
-
+                        var input = form.querySelectorAll("[type=checkbox][name='" + keyval[0] + "']");
                         if( input.length ){
-                            if( input.filter("[type=checkbox],[type=radio]").length ){
-                                input = input.filter( "[value='"+ keyval[1] +"']" ).attr("checked", true);
-                            } else if(input.filter("[type=text]").length ) {
-                                input.val(keyval[1]);
+                            input.forEach(inpt => {
```

Comment:
`inpt` reads like a typo :) Maybe make it `imp` or `i`.

Or just `input` and tweak above as
```js
let checkboxes = form.querySelectorAll("[type=checkbox]...
```